Makefile

```diff
@@ -27,16 +27,13 @@ all: install generate
 develop: install generate-proto
 
 # pip install packages
-install: uninstall
-	pip3 install git+https://github.com/icon-project/icon-service.git@master
-	pip3 install git+https://github.com/icon-project/icon-commons.git@master
-	pip3 install git+https://github.com/icon-project/icon-rpc-server.git@master
-	pip3 install tbears
+install:
+	pip3 install -U git+https://github.com/icon-project/icon-service.git@master
+	pip3 install -U git+https://github.com/icon-project/icon-commons.git@master
+	pip3 install -U git+https://github.com/icon-project/icon-rpc-server.git@master
+	pip3 install -U tbears
 	pip3 install -e .
```

Comment:
Doesn't it need '-U' option?

Reply:
The -U option allows you to install the library regardless of whether the version or some of the files of the library has changed or not. (In case the master version has not changed and some files have been modified)